doc/classes/Joint2D.xml

```diff
@@ -74,12 +74,16 @@
 	</methods>
 	<members>
 		<member name="bias" type="float" setter="set_bias" getter="get_bias">
+			The force with which nodes A and B stay in positional relation to each other.
 		</member>
 		<member name="disable_collision" type="bool" setter="set_exclude_nodes_from_collision" getter="get_exclude_nodes_from_collision">
+			Decides, if [member node_a] and [member node_b] should be able to collide.
```

Comment:
See [Writing Guidelines](http://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html): boolean members have a standard wording.  Finally, it's helpful to show the default value:

    If [code]true[/code] [member node_a] and [member node_b] can collide. Default value: [code]false[/code].